ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsCollector.scala

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool.metrics
+
+import akka.actor.typed.scaladsl.{Behaviors, TimerScheduler}
+import akka.actor.typed.{ActorRef, Behavior}
+import com.daml.ledger.api.benchtool.util.{MetricReporter, TimeUtil}
+
+import java.time.Instant
+import scala.concurrent.duration._
+
+object MetricsCollector {
```

Comment:
This is the old `MetricsManager`. The logic in this class hasn't change - the huge diff is due to renaming.